src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
The `href` attribute in the `<link>` tag should point to a CSS file, not a SCSS file. SCSS files need to be compiled to CSS before they can be linked in an HTML document. Please ensure the `href` points to the correct CSS file.